src/TopBar/index.tsx

```diff
@@ -14,6 +14,7 @@ const propTypes = {
   className: PropTypes.string,
   logoHref: PropTypes.string.isRequired,
   title: PropTypes.node,
+  onLogoClick: PropTypes.func,
```

Comment:
How motivated do you feel about converting these PropTypes to TypeScript? 😉

Reply:
I'd love to but last time I tried the linter complained so I reverted it. Unclear if that's still the case? I wrote this with types originally and then deleted them when I was writing the TopBar component.